space-cli/cmd/modules/addons/database.go

```diff
@@ -106,10 +145,69 @@ func addDatabase(dbtype, username, password, alias, version string) error {
 		return utils.LogError("Could not save hosts file after updating add on containers", err)
 	}
 
-	utils.LogInfo(fmt.Sprintf("Started database (%s) with alias (%s)", dbtype, alias))
+	if autoApply {
+		connDefault := ""
+		switch dbtype {
+		case "postgres":
+			connDefault = fmt.Sprintf("postgres://postgres:mysecretpassword@%s:5432/postgres?sslmode=disable", hostName)
```

Comment:
We might change the default password

Reply:
what about username ?

Reply:
It can change for a few databases. Why don't you have a look at how we start a docker image?